bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -1909,6 +1909,8 @@ lnd/clm2/surfdata_esmf/ctsm5.2.0/surfdata_ne3np4.pg3_hist_1850_78pfts_c240216.nc
 lnd/clm2/surfdata_esmf/ctsm5.2.0/surfdata_C96_hist_1850_78pfts_c240216.nc</fsurdat>
 <fsurdat hgrid="1x1_smallvilleIA" sim_year="1850" >
 lnd/clm2/surfdata_esmf/ctsm5.2.0/surfdata_1x1_smallvilleIA_hist_1850_78pfts_c240221.nc</fsurdat>
+<fsurdat hgrid="1x1_cidadinhoBR" sim_year="2000" >
+lnd/clm2/surfdata_esmf/surfdata_1x1_cidadinhoBR_hist_2000_78pfts_c240613.nc</fsurdat>
```

Comment:
Let's move this into lnd/clm2/testdata to go along with the move of the smallville data. I'll add this to #2515. I also see that the subset_data isn't keeping a useful way to track what version of the model was used to create so added issue #2669 for that.